client/components/jetpack/card/jetpack-product-card-alt-2/features-product-slide-out.tsx

```diff
@@ -32,48 +35,58 @@ const FeaturesProductSlideOut: FunctionComponent< Props > = ( {
 	productBillingTerm,
 	onProductClick,
 } ) => {
+	const {
+		iconSlug,
+		productSlug,
+		displayName,
+		shortName,
+		description,
+		displayTerm,
+		displayPrice,
+		displayCurrency,
+		monthlyProductSlug,
+	} = product;
+
 	const siteId = useSelector( getSelectedSiteId );
 	const currencyCode = useSelector( getCurrentUserCurrencyCode );
+	const purchases = useSelector( ( state ) => getSitePurchases( state, siteId ) );
+	const sitePlan = useSelector( ( state ) => getSitePlan( state, siteId ) );
+	const { originalPrice } = useItemPrice( siteId, product, monthlyProductSlug || '' );
 	const translate = useTranslate();
 
-	// Calculate the product price.
-	const { originalPrice, discountedPrice } = useItemPrice(
-		siteId,
-		product,
-		product?.monthlyProductSlug || ''
-	);
-	const isDiscounted = isFinite( discountedPrice );
-	const productPrice = isDiscounted ? discountedPrice : originalPrice;
 	const formattedPrice = formatCurrency(
-		( ( product && product.displayPrice ) || productPrice ) as number,
-		( ( product && product.displayCurrency ) || currencyCode ) as string,
+		( displayPrice || originalPrice ) as number,
+		( displayCurrency || currencyCode ) as string,
 		{
 			precision: 0,
 		}
 	);
-	const billingTimeFrame = durationToText(
-		( product && product.displayTerm ) || productBillingTerm
-	);
-
-	const slideOutButtonLabel = translate( 'Get {{name/}} %(price)s', {
-		args: {
-			price: formattedPrice,
-		},
-		components: {
-			name: <>{ product?.shortName }</>,
-		},
-	} );
+	const isItemPlanFeature = !! ( sitePlan && planHasFeature( sitePlan.product_slug, productSlug ) );
+	const purchase = isItemPlanFeature
+		? getPurchaseByProductSlug( purchases, sitePlan?.product_slug || '' )
+		: getPurchaseByProductSlug( purchases, productSlug );
+	const slideOutButtonLabel = isItemPlanFeature
+		? translate( 'Manage Subscription' )
+		: translate( 'Get {{name/}} %(price)s', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **8** times:
`translate( 'Get {{name/}} $%(price)s' )` *ES Score: 6*